packages/yswaps/scripts/execute-mainnet-pending-trades.ts

```diff
@@ -128,17 +135,19 @@ async function main() {
 
         const blockProtection = await ethers.getContractAt(BlockProtectionABI, '0xCC268041259904bB6ae2c84F9Db2D976BCEB43E5', ymech);
 
-        // await generateAndSendBundle({
-        //   blockProtection,
-        //   wallet: ymech,
-        //   executeTx,
-        //   gasParams: {
-        //     ...gasPriceParams,
-        //     // gasLimit: confirmedTx.gasUsed.add(confirmedTx.gasUsed.div(5)),
-        //     gasLimit: BigNumber.from(2_000_000),
-        //   },
-        //   nonce,
-        // });
+        console.timeEnd('[Execution] Total trade execution time');
```

Comment:
maybe change text to `Total trade execution time IN FORK` so is more explicit. Could lead to confusion that we are talking about the real trade